### PR TITLE
local env var overrides fabric env issue

### PR DESCRIPTION
**Pull request by jyejare**
Local 'env' variable replacing / overriding fabric 'env'.
Renamed local 'env' to 'envr'

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1931,10 +1931,10 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
     version = distro_info()[1]
     details = os.environ.get('CAP_SUB_DETAILS')
     if details is not None:
-        cv_name, env, ak_name = [item.strip() for item in details.split(',')]
+        cv_name, envr, ak_name = [item.strip() for item in details.split(',')]
```
Comment (abalakh) [inline automation_tools/__init__.py]: why not `env_name` to consist with 2 others? or it's used too?
and btw, in `envr`, what does `r` stand for?

**Comment by abalakh** [comment]
ACK

File: automation_tools/__init__.py (hunk 1 above)
Comment (jyejare) [inline automation_tools/__init__.py]: Sounds good 'env_name', I was not getting a good name and so 'envr' :)

**Comment by sthirugn** [comment]
ACK

**Pull request merged**